exercises/practice/accumulate/description.md

```diff
@@ -0,0 +1,26 @@
+# Description
```

Comment:
This file should be removed, as exercises don't have a description.md file (the new file is `.docs/instructions.md`, which you've added).